libs/components/indicators/src/lib/modules/tokens/tokens.component.scss

```diff
@@ -6,7 +6,7 @@ $sky-tokens-gutter-size: 2px;
   display: flex;
   flex-wrap: wrap;
   align-items: baseline;
-  margin: 0 -$sky-tokens-gutter-size;
+  margin: -1 * $sky-tokens-gutter-size;
```

Comment:
Should this be `0 (-1 * $sky-tokens-gutter-size)`?

Reply:
I don't know if that was the intent, but the design we've been using has been `(0 - $sky-tokens-gutter-size)` and we see warnings about the ambiguity. I tried `0 -1 * $sky-tokens-gutter-size` but that added vertical space; the current rule computes to `margin: -2px`.

We have several places where we're negating a variable and those will break in a future version.

I can remove this change as it's not really related to the issue.

Reply:
Strange; I'm guessing the original intent for that statement was `0 -2px`, but it looks like it evaluates to `-2px` (like you mentioned). 🤷 We'll keep your change for now.